mmtrack/models/trackers/byte_tracker.py

```diff
@@ -73,8 +73,11 @@ def init_track(self, id, obj):
         bbox = bbox_xyxy_to_cxcyah(self.tracks[id].bboxes[-1])  # size = (1, 4)
         assert bbox.ndim == 2 and bbox.shape[0] == 1
         bbox = bbox.squeeze(0).cpu().numpy()
+        label_idx = self.memo_items.index('labels')
+        obj_label = obj[label_idx]
         self.tracks[id].mean, self.tracks[id].covariance = self.kf.initiate(
             bbox)
+        self.tracks[id].label = obj_label
```

Comment:
Why do you need to reset `self.tracks[id].label`?
The `self.tracks[id].label` is set in `super().init_track(id, obj)`.
I think `super().init_track(id, obj)` is always equal to `obj_label`.

Reply:
I see, I didn't notice this. I have removed this duplicated `self.tracks[id].label` in the new commit.